<?xml version="1.0"?>
<launch>
 	<!-- <arg name="ttc_th" default="0.3" /> -->

	<!-- file="$(find f1tenth_simulator)/launch/simulator.launch"> -->	
	<include
		file="$(find racecar)/launch/teleop.launch">
 	</include>


 
	<!-- <rosparam command="load" file="$(find f1tenth_simulator)/params.yaml"/> -->
		 	<!-- <rosparam command="load" file="$(find racecar)/params.yaml"/> -->
	 	<!-- <param name="brake_ttc_threshold" value="$(arg ttc_th)"/> -->
 	<node 
		pkg="safety_node1" 
	 	name="safety_node" 
	 	type="safety_node" 
	 	output="screen">

	 	<remap from="/scan" to="/scan"/>
		<remap from="/odom" to="/vesc/odom"/>
		<remap from="/brake_bool" to="/brake_bool"/>
		<remap from="/brake" to="/vesc/low_level/ackermann_cmd_mux/input/safety"/>
		<remap from="/vesc/joy" to="/joy"/>
		<remap from="/nav" to="/vesc/high_level/ackermann_cmd_mux/input/nav_0"/>
	 </node>
	 
</launch>
	
